### DISCUSSION TITLE
What's the wrong with my code?

### DISCUSSION THREAD

**Discussion by devslayertech** [quasarframework/app-extension-qiconpicker#171] [Q&A] [answered]
Since the update from Quasar v1 to Quasar v2 I was stuck for a while and now I can't load and use the IconPicker anymore. 

Everything seems to have gotten more tiring and boring with this update.

**I received the error:**
![image](https://user-images.githubusercontent.com/100967476/177918631-fc3c19ab-ef0d-440e-aabe-62fc9a579fd9.png)

The result in browser:
![image](https://user-images.githubusercontent.com/100967476/177915434-cae9f053-52f3-4a44-b320-2409a1d15aa0.png)

My Code:

```HTML
<template>
  <q-page class="row items-center justify-evenly">
    <div class="q-pa-lg flex flex-center">
      <div style="width: 700px;">
        <q-input
          v-model="data.paginationColor"
          filled
          label="Pagination Color"
          hint="Enter colors only from the Quasar color palette (ex: 'orange-8')"
          class="q-mb-md"
        />
        <q-separator />
        <q-icon-picker
          v-model="data.value"
          v-model:model-pagination="data.pagination"
          icon-set="material-icons"
          :pagination-props="{
            maxPages: 5,
            input: true,
            color: data.paginationColor
          }"
          style="height: 220px;"
        />
      </div>
    </div>
  </q-page>
</template>

<script>
import { defineComponent, ref } from 'vue'

export default defineComponent({
  name: 'IndexPage',

  setup () {
    const data = ref({
      value: '',
      paginationColor: 'orange-10',
      pagination: {
        itemsPerPage: 60,
        page: 0
      }
    })

    return {
      data
    }
  }
})
</script>
```

quasar.config.js - My extras
![image](https://user-images.githubusercontent.com/100967476/177918066-0de1bee7-644d-4d61-bd5f-260f33b51a9f.png)

I'm using Vite with Typescript

I only created the new Quasar App then installed the QIconPicker and used this code above:
`yarn add @quasar/quasar-ui-qiconpicker`

I'm sorry for my ignorance. I'm learning everything again.

**Comment by devslayertech** [comment]
Editing the line of error in node_modules/@quasar/quasar-ui-qiconpicker/dist/index.esm.js to:

console.error(\`QIconPicker: cannot find icon set found called ${ e }\`);

The error now is:
`QIconPicker: cannot find icon set found called ReferenceError: require is not defined`

**Comment by hawkeye64** [comment, marked as answer]
This App Extension does not work with Vite yet
